### PR TITLE
sessions/bincompat: Update contents to use unikraft-upb/scripts

### PR DESCRIPTION
**Pull request by razvand**
Update to the most recent version of `app-elfloader`, `run-app-elfloader`. Use `scripts` repository in `unikraft-upb`.

### CODE REVIEW

File: content/en/community/hackathons/sessions/bincompat/content/Work-Items_02.-Build-app-elfloader.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+$ ./do.sh clean
+
+$ ./do.sh setup_plain
+
+$ ./do.sh build
+
+$ ./do.sh run_built ...
+```
+
+If, on the other side, you want to more debugging information, use `setup_debug`, such as the commands below:
```
Comment (StefanJum) [inline content/en/community/hackathons/sessions/bincompat/content/Work-Items_02.-Build-app-elfloader.md:30]:
```suggestion
If, on the other side, you want to have more debugging information, use `setup_debug`, such as the commands below:
```

File: content/en/community/hackathons/sessions/bincompat/content/Work-Items_02.-Build-app-elfloader.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+If you want to remove all the pesky debugging information, use `setup_plain`, such as the commands below:
+
+```console
+$ ./do.sh clean
+
+$ ./do.sh setup_plain
+
+$ ./do.sh build
+
+$ ./do.sh run_built ...
```
Comment (StefanJum) [inline content/en/community/hackathons/sessions/bincompat/content/Work-Items_02.-Build-app-elfloader.md:27]: Is `run_built` a command in the [`app-elfloader` script](https://github.com/unikraft-upb/scripts/blob/main/make-based/app-elfloader/do.sh#L205-L251)? It does not seem so.

**Review by StefanJum — COMMENTED** [review]
Just a few comments @razvand.

File: content/en/community/hackathons/sessions/bincompat/content/Work-Items_02.-Build-app-elfloader.md (hunk 2 above)
Comment (razvand) [reply content/en/community/hackathons/sessions/bincompat/content/Work-Items_02.-Build-app-elfloader.md:27]: See this: https://github.com/unikraft-upb/scripts/pull/32

**Review by StefanJum — APPROVED** [review]
All good, thanks.

Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**